docs/api.rst

```diff
@@ -1274,6 +1274,17 @@ to handle it, which defaults to print a traceback and ignoring the exception.
     :param entry: The entry that was created.
     :type entry: :class:`AuditLogEntry`
 
+.. function:: on_application_command(interaction, app_cmd)
```

Comment:
I think it would be simpler if there was no command argument, as that is possible with `interaction.application_command`